### PR TITLE
Bug 559883 - [Passage] PDE Integration : complete licensing scenario

### PR DESCRIPTION
**Pull request by ruspl-afed**
Add core bundle

Signed-off-by: Alexander Fedorov <alexander.fedorov@arsysop.ru>

### CODE REVIEW

File: bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+#
+# SPDX-License-Identifier: EPL-2.0
+#
+# Contributors:
+#     ArSysOp - initial API and implementation
+###############################################################################
+
+Bundle-Name = Passage LDC PDE Core Templates
+Bundle-Vendor = Eclipse Passage
+Bundle-Copyright = Copyright (c) 2021 ArSysOp and others.\n\
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties:17]: No others, ArSysOp only.

File: bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties [hunk 2]
```diff
@@ -0,0 +1,23 @@
+#Properties file for org.eclipse.passage.ldc.pde.core.templates
+###############################################################################
+# Copyright (c) 2019, 2021 ArSysOp and others
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties:3]: Only 2021?
There is no _others_ this time.

File: bundles/org.eclipse.passage.ldc.pde.core.templates/build.properties [hunk 3]
```diff
@@ -0,0 +1,19 @@
+###############################################################################
+# Copyright (c) 2021 ArSysOp and others
```
Comment (eparovyshnaya) [inline bundles/org.eclipse.passage.ldc.pde.core.templates/build.properties:2]: No others.

**Review by eparovyshnaya — APPROVED** [review]
Please take a look at license headers for new bundle.

File: bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties (hunk 1 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties:17]: fair point

File: bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties (hunk 2 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.ldc.pde.core.templates/OSGI-INF/l10n/bundle.properties:3]: fair point

File: bundles/org.eclipse.passage.ldc.pde.core.templates/build.properties (hunk 3 above)
Comment (ruspl-afed) [reply bundles/org.eclipse.passage.ldc.pde.core.templates/build.properties:2]: fair point

**Review by eparovyshnaya — APPROVED** [review]

**Pull request merged**